TESTS/mbedtls/ecp/main.cpp

```diff
@@ -20,8 +20,6 @@
 #include "utest.h"
 #include "rtos.h"
 
-using namespace utest::v1;
-
```

Comment:
If the other ```status_t``` was also in a namespace then using ```using``` on one might be ok. ```using namespace utest::v1``` statement may not be a problem now but can be in future. Hence a maintenance thing.
Secondly, I didn't have partner code to verify impact of moving the ```using```. Hence I went for a more robust approach.